packages/core/src/interpreter.ts

```diff
@@ -151,7 +151,7 @@ export class Interpreter<
    * Whether the service is started.
    */
   public initialized = false;
-  private _status: InterpreterStatus = InterpreterStatus.NotStarted;
+  public _status: InterpreterStatus = InterpreterStatus.NotStarted;
```

Comment:
Should we make this `status` instead then?

Reply:
done